src/pymor/vectorarrays/interfaces.py

```diff
@@ -383,27 +383,39 @@ def lincomb(self, coefficients):
         """
         pass
 
-    def norm(self, product=None):
+    def norm(self, product=None, tol=1e-10, raise_complex=True):
```

Comment:
Could you add `@defaults('tol', 'raise_complex')` above this, to make it like `constructions.induced_norm`?